src/components/RichTextContainer.ts

```diff
@@ -125,7 +125,8 @@ export default class RichTextContainer extends Component<RichTextContainerProps,
         }
     }
 
-    private handleOnChange(value: string) {
+    private handleOnChange(value: string, event: KeyboardEvent) {
+        event.cancelBubble = true;
```

Comment:
Why not use `stopPropagation` ?
https://developer.mozilla.org/en-US/docs/Web/API/Event/cancelBubble

Reply:
Done